wbooks/src/app/screens/Library/index.tsx

```diff
@@ -3,8 +3,8 @@ import { SafeAreaView, FlatList, ListRenderItem } from 'react-native';
 import { useDispatch, useSelector } from 'react-redux';
 import { Book, BookState } from '@interfaces/Book';
 import BookActions from '@redux/book/actions';
+import BookComponent from '@components/Book';
```

Comment:
Extract `BooksList`, not `BookComponent`

Reply:
I extracted BookList, but BookComponent is a common component between Library and Filter, so I think that this extraction is okay, let me know what dow you think.